### DISCUSSION TITLE
How to stop Ruffle from creating new instances

### DISCUSSION THREAD

**Discussion by ErzenXz** [ruffle-rs/ruffle#10623] [General]
Hello so im making a games website, but have a problem, if the user browser supports flash, i want to show the flash content and not use ruffle, i tried data-ruffle-skip but it did not work.

Can you help me?

**Comment by n0samu** [comment]
There is no such thing as `data-ruffle-skip`. I think you would need to write some JavaScript code yourself that checks for the presence of Flash in `navigator.plugins` and inserts the Ruffle script tag only if the plugin is not present.

**Reply by n0samu** [reply]
... or you can use this library for it: https://github.com/tchakabam/FlashDetect
